label_studio_converter/converter.py

```diff
@@ -851,20 +867,52 @@ def convert_to_yolo(
                         or 'rectangle' in label
                         or 'labels' in label
                     ):
-                        xywh = self.rotated_rectangle(label)
-                        if xywh is None:
-                            continue
+                        if is_obb:
+                            base_y = label["original_height"]
```

Comment:
I would replace it to call get_rotated_corner_points from my code, it's a bit more clear and computationally optimized.